components/TasksTable.js

```diff
@@ -40,7 +40,7 @@ function TasksTable() {
       <Text style={styles.title}>Tasks</Text>
       {!tasks || tasks.length === 0 || tableContents.length === 0 ? (
         <View style={{ paddingBottom: 16 }}>
-          <Text>No upcoming tasks for this semester.</Text>
+          <Text style={styles.text}>No upcoming tasks for this semester.</Text>
```

Comment:
I merged Isa's feature earlier this week, did you get that? Because I don't see the pagination stuff